geojson2csv/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7-alpine
+
+LABEL "name"="geojson2csv"
+LABEL "maintainer"="Unacast <developers+github@unacast.com>"
+LABEL "version"="1.0.0"
+
+LABEL "com.github.actions.name"="Geojson2csv"
+LABEL "com.github.actions.description"="Convert geojson to csv"
+LABEL "com.github.actions.icon"="truck"
+LABEL "com.github.actions.color"="green"
+
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+
+RUN apk add --no-cache \
+gcc \
+libc-dev \
+geos-dev \
+curl
+
+COPY requirements.txt /
```

Comment:
These two line could be replaced by `pip install geojson shapely` I believe

Reply:
Moved `pip install` to dockerfile